README.md

```diff
@@ -152,6 +153,13 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 * [Apache HBase](http://hbase.apache.org) - Apache HBase is the Hadoop database, a distributed, scalable, big data store.
 * [Infinispan](http://infinispan.org/) -  Distributed and highly concurrent key/value datastore used for caching.
 
+## Distributed Deployment
```

Comment:
Is it really only deployment? I would say it's kind of cluster management software, isn't it?